styles.css

```
@@ -12,32 +14,358 @@
     5. Media Queries
 ////////////////////////*/
 
-
-
 /* ------------------------------- 1. Global Styles -------------------------------- */
 
-
 * {
     margin: 0;
     box-sizing: border-box;
 }
 
+img {
+    overflow-clip-margin: border-box;
+    margin: 0;
+}
+
+:root {
+    --main-text-color:#0C0C0C;
+    --main-floating-words:#F2F0FF;
+    --main-background-projects:#F3F6F9;
+    --main-service-rule-text:#7A7A7A;
+    --purple: #5722cd;
+    --main-purple: radial-gradient(circle, rgba(87,34,205,1) 39%, rgba(75,73,232,1) 93%);
+    --pink:#ED4385;
+}
+
 /* ------------------------------- 2. Typography ----------------------------------- */
 
+.dev {
+    border: 2px solid black;
+}
+
 body {
-    font-family:
+    font-family: 'Merriweather' , 'Roboto', sans-serif;
+
+}
+
+ h2 {
+    color: var(--purple);
+    margin: 0 0 10px 0;
+     font-family: Merriweather, sans-serif;
+}
+.section-wrapper h2 {
+    color: var(--main-text-color);
+}
+
+footer h2{
+    color: var(--main-floating-words);
+}
+
+footer p {
+    color: var(--main-floating-words)
+}
+
+footer form {
+    color: var(--main-floating-words)
+}
+
+.inner-container__introduction a {
+    font-weight: 400;
+    letter-spacing: normal;
+}
+
+nav a {
+    font-weight: 400;
+    letter-spacing: normal;
+}
+
+a {
+    color: var(--purple);
+    font-family: Roboto, sans-serif;
+    font-weight: 600;
+    letter-spacing: .2rem;
+}
+
+p {
+    margin: 20px 0 20px 0;
```

Dit kun je wat korter schrijven
```suggestion
    margin: 20px 0;
```